projects/plugins/jetpack/class.jetpack-cli.php

```diff
@@ -1178,6 +1178,8 @@ public function partner_cancel( $args, $named_args ) {
 	 * : Overrides the siteurl option via the site_url filter, or the WP_SITEURL constant
 	 * [--partner_tracking_id=<partner_tracking_id>]
 	 * : This is an optional ID that a host can pass to help identify a site in logs on WordPress.com
+	 * [--partner_id=<partner_id>]
```

Comment:
This would require that partners change how they call the script for us to then have the information to add to the register.

My suggestion would be that we probably:

1. Remove the change in this `class.jetpack-cli.php` file
2. Hard-code the `from` argument that is sent from `class.jetpack-provision.php` to something like `from=jetpack-start

Then, on the backend of WP.com, we can map that to the `external` referral source.